packages/time-picker/test/time-picker.test.js

```diff
@@ -432,28 +432,28 @@ describe('time-picker', () => {
       expect(timePicker.max).to.be.equal('19:00');
     });
 
-    it('should not allow setting a value lower than min property value', () => {
+    it('should not adjust value lower than min set using property', () => {
```

Comment:
How about we remove these tests altogether? If someone adds a feature like that in the future, it will likely be intentional.

Reply:
Good point, I agree. Let's drop them altogether 👍

Reply:
Sure, that makes sense.

Reply:
Done.